pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2021,6 +2021,7 @@
     private String replicatorPrefix = "pulsar.repl";
     @FieldContext(
         category = CATEGORY_REPLICATION,
+        dynamic = true,
```

Comment:
Do we need broker restart for this to take effect?

Reply:
IMO, no need to restart.

Reply:
Yes, It won't apply to existing replicators.
But I understand that existing replicators can also take effect by deleting and rebuilding existing replicators.

Reply:
At least the newly created replicators can take effect immediately

Reply:
> At least the newly created replicators can take effect immediately

I think we need to add this in the doc.

Reply:
OK   I will  fix

Reply:
> > At least the newly created replicators can take effect immediately
> 
> I think we need to add this in the doc.

like this?
        doc = "Replicator producer queue size. "
                + "When dynamically modified, it only takes effect for the newly added replicators"